app/services/listing_index_service/api/listings.rb

```diff
@@ -31,6 +28,7 @@ def search(community_id:, search:, includes: [])
           listings: res[:listings].map { |search_res|
             search_res.merge(url: "#{search_res[:id]}-#{search_res[:title].to_url}")}))
       }.or_else {
+        raise search_result.data if Rails.env.development?
```

Comment:
Same comment here as below: This probably would be cleaner if you'd pass `raise_errors` as a variable and not depend on the outside world state.